### ISSUE TITLE
No CUDA GPU found in your device

### ISSUE THREAD

**Issue by nekohacker591** [yangheng95/SuperResolutionAnimeDiffusion#3]
![image](https://user-images.githubusercontent.com/73713314/221443205-c162121b-6c51-469f-86f7-fe322cd59bb6.png)
![image](https://user-images.githubusercontent.com/73713314/221443218-a8909257-29e4-4575-858c-22901c52b95a.png)
![image](https://user-images.githubusercontent.com/73713314/221443228-d4510b9f-58d3-4107-a6de-400d24addeb2.png)

**Comment by yangheng95** [comment]
Please try input `nvidia-smi` in your cmd, if it does not work, you may need to fix cuda by reinstallation.

**Comment by nekohacker591** [comment]
| NVIDIA-SMI 528.49       Driver Version: 528.49       CUDA Version: 12.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name            TCC/WDDM | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA GeForce ... WDDM  | 00000000:01:00.0 Off |                  N/A |
| N/A   60C    P8     6W /  90W |     51MiB /  8192MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A     16700    C+G   ...IA GeForce Experience.exe    N/A      |
+-----------------------------------------------------------------------------+

**Comment by nekohacker591** [comment]
also do you have a discord for live chat

**Comment by nekohacker591** [comment]
if you have a discord add bad food#3946

**Comment by yangheng95** [comment]
Please check if you install the pytorch with cuda bundle, use the following code:
```python3
import torch
print(torch.cuda.is_available())
```
If you install the cpu version, please reinstall pytorch from: https://pytorch.org/get-started/locally/

**Comment by nekohacker591** [comment]
is this correct
conda install pytorch torchvision torchaudio pytorch-cuda=11.6 -c pytorch -c nvidia

**Comment by yangheng95** [comment]
Yes if you use conda, otherwise if you use pip, you can use: pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu116
